assets/scripts/segments/SegmentCanvas.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { getSegmentVariantInfo } from './info'
+import { drawSegmentContents, getVariantInfoDimensions } from './view'
+import { TILE_SIZE } from './constants'
+
+const WIDTH_PALETTE_MULTIPLIER = 4 // Dupe from palette.js
```

Comment:
We could take this opportunity to move `WIDTH_PALETTE_MULTIPLIER` to a `./constants.js` file instead of duping it (it was a hack to get around the need to import a file that ran side effects).